### PR TITLE
iso-646 fixes for CUDA / CL / TBB / Ptex

### PR DESCRIPTION
**Pull request by davidgyu**

### CODE REVIEW

**Comment by c64kernal** [comment]
Thanks David!  In the file tutorials/far/tutorial_1/hbr_to_vtr.h, the last diff is '&' instead of '&&'.

File: tutorials/far/tutorial_1/hbr_to_vtr.h [hunk 1]
```diff
@@ -408,7 +408,7 @@ FarTopologyRefinerFactory<OsdHbrConverter>::assignComponentTopology(
                 virtual void operator() (OsdHbrHalfedge &e) {
 
                     OsdHbrFace * f=e.GetFace();
```
Comment (thomthom) [inline tutorials/far/tutorial_1/hbr_to_vtr.h]: Should be: &&

**Comment by thomthom** [comment]
Thanks for taking on these. I did not have the environment set up to build these.

File: tutorials/far/tutorial_1/hbr_to_vtr.h (hunk 1 above)
Comment (davidgyu) [inline tutorials/far/tutorial_1/hbr_to_vtr.h]: Yes, thanks for the careful reviews!

**Pull request merged**